x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/planner/LocalExecutionPlannerTests.java

```diff
@@ -107,6 +107,21 @@ public void testLuceneTopNSourceOperator() throws IOException {
         assertThat(factory.limit(), equalTo(10));
     }
 
+    public void testLuceneTopNSourceOperatorDistanceSort() throws IOException {
+        int estimatedRowSize = randomEstimatedRowSize(estimatedRowSizeIsHuge);
+        FieldAttribute sortField = new FieldAttribute(Source.EMPTY, "point", new EsField("point", DataType.GEO_POINT, Map.of(), true));
+        EsQueryExec.GeoDistanceSort sort = new EsQueryExec.GeoDistanceSort(sortField, Order.OrderDirection.ASC, 1, -1);
```

Comment:
nit: we could randomize the sort a little for good measure (at least the sort direction between ASC and DESC)